netdata-installer.sh

```diff
@@ -698,226 +698,230 @@ run find "${NETDATA_WEB_DIR}" -type d -exec chmod 0775 {} \;
+      if [ "$KEY" = "$ARCH" ]; then
+        ARCH="${VALUE}"
+        break
+      fi
+    done
+    tmp=$(mktemp -d /tmp/netdata-go-XXXXXX)
+    GO_PACKAGE_BASENAME="go.d.plugin-${GO_PACKAGE_VERSION}.${OS}-${ARCH}.tar.gz"
+
+    if [ -z "${NETDATA_LOCAL_TARBALL_OVERRIDE_GO_PLUGIN}" ]; then
+      download_go "https://github.com/netdata/go.d.plugin/releases/download/${GO_PACKAGE_VERSION}/${GO_PACKAGE_BASENAME}" "${tmp}/${GO_PACKAGE_BASENAME}"
+    else
+      progress "Using provided go.d tarball ${NETDATA_LOCAL_TARBALL_OVERRIDE_GO_PLUGIN}"
+      run cp "${NETDATA_LOCAL_TARBALL_OVERRIDE_GO_PLUGIN}" "${tmp}/${GO_PACKAGE_BASENAME}"
+    fi
+
+    if [ -z "${NETDATA_LOCAL_TARBALL_OVERRIDE_GO_PLUGIN_CONFIG}" ]; then
+      download_go "https://github.com/netdata/go.d.plugin/releases/download/${GO_PACKAGE_VERSION}/config.tar.gz" "${tmp}/config.tar.gz"
+    else
+      progress "Using provided config file for go.d ${NETDATA_LOCAL_TARBALL_OVERRIDE_GO_PLUGIN_CONFIG}"
+      run cp "${NETDATA_LOCAL_TARBALL_OVERRIDE_GO_PLUGIN_CONFIG}" "${tmp}/config.tar.gz"
+    fi
+
+    if [ ! -f "${tmp}/${GO_PACKAGE_BASENAME}" ] || [ ! -f "${tmp}/config.tar.gz" ] || [ ! -s "${tmp}/config.tar.gz" ] || [ ! -s "${tmp}/${GO_PACKAGE_BASENAME}" ]; then
+      run_failed "go.d plugin download failed, go.d plugin will not be available"
+      echo >&2 "Either check the error or consider disabling it by issuing '--disable-go' in the installer"
+      echo >&2
+      return 0
+    fi
+
+    grep "${GO_PACKAGE_BASENAME}\$" "${INSTALLER_DIR}/packaging/go.d.checksums" > "${tmp}/sha256sums.txt" 2> /dev/null
+    grep "config.tar.gz" "${INSTALLER_DIR}/packaging/go.d.checksums" >> "${tmp}/sha256sums.txt" 2> /dev/null
+
+    # Checksum validation
+    if ! (cd "${tmp}" && safe_sha256sum -c "sha256sums.txt"); then
+
+      echo >&2 "go.d plugin checksum validation failure."
+      echo >&2 "Either check the error or consider disabling it by issuing '--disable-go' in the installer"
+      echo >&2
+
+      run_failed "go.d.plugin package files checksum validation failed."
+      return 0
+    fi
+
+    # Install new files
+    run rm -rf "${NETDATA_STOCK_CONFIG_DIR}/go.d"
+    run rm -rf "${NETDATA_STOCK_CONFIG_DIR}/go.d.conf"
+    run tar -xf "${tmp}/config.tar.gz" -C "${NETDATA_STOCK_CONFIG_DIR}/"
+    run chown -R "${ROOT_USER}:${NETDATA_GROUP}" "${NETDATA_STOCK_CONFIG_DIR}"
+
+    run tar xf "${tmp}/${GO_PACKAGE_BASENAME}"
+    run mv "${GO_PACKAGE_BASENAME/\.tar\.gz/}" "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/go.d.plugin"
+    if [ "${UID}" -eq 0 ]; then
+      run chown "root:${NETDATA_GROUP}" "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/go.d.plugin"
+    fi
+    run chmod 0750 "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/go.d.plugin"
+    rm -rf "${tmp}"
+  fi
+  return 0
 }
 install_go
 
 # -----------------------------------------------------------------------------
 progress "Telemetry configuration"
 
+if [ x"$DO_NOT_TRACK" = x"1" ]; then
```

Comment:
Why not simply `if [ "${DO_NOT_TRACK}" = "1" ]; then` ?

```suggestion
if [ "$DO_NOT_TRACK" = "1" ]; then
```